summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java

```diff
@@ -82,6 +82,10 @@ public Void apply(IN v) {
     return new Service<Memory, JMemoryService<K, V>, K, V>(new JMemoryService(service));
   }
 
+<<<<<<< HEAD
+=======
+  private Memory platform = new Memory(new JobId("inMem"));
```

Comment:
was this deleted on develop on purpose, or added in master and we want to keep it?